docs/ops/arithmetic/Sinh_1.md

```diff
@@ -6,28 +6,28 @@
 
 **Short description**: *Sinh* performs element-wise hyperbolic sine (sinh) operation with given tensor.
 
-**Attributes**:
+**Detailed description**: *Sinh* does the following with the input tensor *a*:
+
+\f[
+a_{i} = sinh(a_{i})
+\f]
+
+**Attributes**: 
 
     No attributes available.
 
 **Inputs**
 
-* **1**: An tensor of type T. **Required.**
+* **1**: An tensor of type T and arbitrary shape. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise sinh operation. A tensor of type *T*.
+* **1**: The result of element-wise sinh operation. A tensor of type T and the same shape as input.
 
 **Types**
 
 * *T*: any numeric type.
 
-*sinh* does the following with the input tensor *a*:
-
-\f[
-a_{i} = sinh(a_{i})
-\f]
-
 **Examples**
```

Comment:
Please consider to address this comment here: https://github.com/openvinotoolkit/openvino/pull/6429#discussion_r660415923